spark-3.5-scala-2.13.yaml

```diff
@@ -26,18 +26,20 @@ environment:
       - glibc-locale-en
       - grep
       - maven
-      - openjdk-17-default-jdk
+      - openjdk-17
       - perl-utils
       - procps
+      - py3-setuptools
       - py3.11-pip
-      - python-3.11
+      - python3
```

Comment:
We should be explicit with the Python version in case there's drift between what we build with and what's fetched at runtime

Reply:
Thanks, fixed